index.js

```diff
@@ -1,5 +1,10 @@
-const { join } = require('path');
+import { join } from "path";
 
-module.exports = {
-  iconsPath: join(__dirname, 'dist'),
+// Since __dirname no longer exists with ESM, https://stackoverflow.com/a/50052194
+import { dirname } from "path";
+import { fileURLToPath } from "url";
+export const __dirname = dirname(fileURLToPath(import.meta.url));
```

Comment:
`__dirname` doesn't work as expected in esm, so we need this

Reply:
The recommended way to do this by the node.js doc is as follow:

```js
const absolute_filepath = new URL('../a/path/to/a/file.js', import.meta.url);
```

The above will give you a URL object which node.js file APIs understand but if you need a absolute path to a file as a string one can do as so:

```js
const absolute_filepath = new URL('../a/path/to/a/file.js', import.meta.url).pathname;
```